soyuz_project/settings/local.py

```diff
@@ -4,7 +4,7 @@
 # GENERAL
 # ------------------------------------------------------------------------------
 # https://docs.djangoproject.com/en/dev/ref/settings/#debug
-DEBUG = True
+DEBUG = False
```

Comment:
I don't think you need to change this?

Reply:
i changed it because i wanted to see the error pages, or else it would show the django error page:
`You’re seeing this error because you have DEBUG = True in your Django settings file. Change that to False, and Django will display a standard 404 page.`